crates/bitcoind_rpc/Cargo.toml

```diff
@@ -18,11 +18,11 @@ workspace = true
 [dependencies]
 bitcoin = { version = "0.32.0", default-features = false }
 bitcoincore-rpc = { version = "0.19.0" }
-bdk_core = { path = "../core", version = "0.1", default-features = false }
+bdk_core = { path = "../core", version = "0.2", default-features = false }
 
 [dev-dependencies]
 bdk_testenv = { path = "../testenv", default-features = false }
-bdk_chain = { path = "../chain", version = "0.18" }
+bdk_chain = { path = "../chain" }
```

Comment:
Was removing the version intentional? The other crates are still using it.

Reply:
Yes it was intentional since we don't need to put the version for `dev-dependencies` and it's easier to maintain/release without them.